src/DataRepo.js

```diff
@@ -0,0 +1,95 @@
+import Trip from "./Trips";
+import Traveler from "./Traveler";
+import Destination from "./Destination";
+import dayjs from "dayjs";
+import { tripsSampleData } from "../test/Sample-trips-data";
+import travelerSampleData from "../test/Sample-traveler-data";
+import destinationSampleData from "../test/Sample-destination-data";
+
+class DataRepo {
+    constructor(data){
+        this.travelers = data.travelers
+        this.trips = data.trips
+        this.destinations = data.destinations
+        this.currentTraveler = new Traveler()
+        this.date = dayjs().format('YYYY/MM/DD')
+    }
+    returnCurrentTravelerById(id){
+        let travlerInfo = this.travelers.find(traveler => id === traveler.id)
+        this.currentTraveler = travlerInfo
+        return travlerInfo
+    }
+    returnCurrentTravelerFirstName() {
+        let firstName = this.currentTraveler.name.split(' ')[0]
+        return firstName
+    }
+    returnDestinationById(id){
+        let destFilter = this.destinations.find(dest => id === dest.id)
+        return destFilter
+    }
+    returnTripsForCurrentTraveler(id){
+        return this.trips.filter(trip =>  {
+            if (trip.userID === id){
```

Comment:
Don't forget- when using a .filter() , you don't need an if statement, it's implied in the conditional . Maybe refactor to remove.